hypervisor/lib/crypto/mbedtls/sha256.c

```diff
@@ -188,10 +187,13 @@ int32_t mbedtls_internal_sha256_process( mbedtls_sha256_context *ctx,
         P( A[3], A[4], A[5], A[6], A[7], A[0], A[1], A[2], R(i+5), K[i+5] );
         P( A[2], A[3], A[4], A[5], A[6], A[7], A[0], A[1], R(i+6), K[i+6] );
         P( A[1], A[2], A[3], A[4], A[5], A[6], A[7], A[0], R(i+7), K[i+7] );
+
```

Comment:
Why do this change ?

Reply:
" for( i = 0; i < 16; i += 8 )" violate  "271S: For loop incrementation is not simple."